src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -29,10 +29,8 @@ ov::SoPtr<ov::ITensor> make_tensor_slice(ov::SoPtr<ov::ITensor> tensor,
 }
 }  // anonymous namespace
 
-ov::npuw::LLMInferRequest::LLMInferRequest(const std::shared_ptr<ov::npuw::LLMCompiledModel>& compiled_model,
-                                           const ov::npuw::LLMCompiledModel::KVCacheDesc& kvcache_desc)
-    : ov::ISyncInferRequest(compiled_model),
-      m_kvcache_desc(kvcache_desc) {
+ov::npuw::LLMInferRequest::LLMInferRequest(const std::shared_ptr<ov::npuw::LLMCompiledModel>& compiled_model)
+    : ov::ISyncInferRequest(compiled_model) {
     m_kvcache_request = compiled_model->m_kvcache_compiled->create_infer_request();
```

Comment:
Couldn't fine the place where `m_npuw_llm_compiled_model` is initialized:
https://github.com/openvinotoolkit/openvino/blob/b52da47c96a527a92debcf099d5ee5f5b6737137/src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.hpp#L46

Reply:
Removed!